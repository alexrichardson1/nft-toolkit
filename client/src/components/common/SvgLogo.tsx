interface PropsT {
  icon: string;
  width: string | number;
  height: string | number;
}

const SvgLogo = (props: PropsT): JSX.Element => (
  <img width={props.width} height={props.height} src={props.icon} />
);

export default SvgLogo;
