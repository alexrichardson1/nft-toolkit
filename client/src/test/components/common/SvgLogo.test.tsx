import SvgLogo from "components/common/SvgLogo";
import { mount } from "enzyme";
import AvalancheLogo from "images/avalanche-logo.svg";

test("SvgLogo snapshot", () => {
  const tree = mount(
    <SvgLogo alt="test" icon={AvalancheLogo} width="50" height="50" />
  );
  expect(tree).toMatchSnapshot();
});
