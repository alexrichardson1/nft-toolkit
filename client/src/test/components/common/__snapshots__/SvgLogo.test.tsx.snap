// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgLogo snapshot 1`] = `
<pure(SvgLogo)
  height="50"
  icon="avalanche-logo.svg"
  width="50"
>
  <SvgLogo
    height="50"
    icon="avalanche-logo.svg"
    width="50"
  >
    <img
      alt="logo"
      height="50"
      src="avalanche-logo.svg"
      style={
        Object {
          "margin": "0",
        }
      }
      width="50"
    />
  </SvgLogo>
</pure(SvgLogo)>
`;
